Source/WebGPU/WebGPU/CommandEncoder.mm

```diff
@@ -126,6 +126,7 @@ static MTLStoreAction storeAction(WGPUStoreOp storeOp)
 {
     if (m_blitCommandEncoder) {
         [m_blitCommandEncoder endEncoding];
+        [m_blitCommandEncoder updateFence:m_device->getQueue().generateFenceToWaitFor()];
```

Comment:
This shouldn't be necessary.

Reply:
> We shouldn't need fences to do this; Metal has automatic hazard tracking. It is correct to use the Blit command encoder to fill a buffer with 0s and then just use it in a subsequent command/pass without any fences.

That was my expectation too but it does not seem to work. Let me take a look with the debugger and see what is occurring